### PR TITLE
shim 'easy' commands to exec instead of embed

### PR DESCRIPTION
**Pull request by deads2k**
I think I'm starting non-controversially.  I'm going to continue exploring the idea, but let's see if there are any complaints thus far.

@smarterclayton @eparis @liggitt @fabianofranz

### CODE REVIEW

**Comment by eparis** [comment]
honestly, I'd rather just kill os cli/adm/etc altogether than cobble in an exec :-(  Less engineering work, less likely to run into unexpected problems down the road, and really, not tons of customers are going to have been using these...

**Comment by deads2k** [comment]
> honestly, I'd rather just kill os cli/adm/etc altogether than cobble in an exec :-( Less engineering work, less likely to run into unexpected problems down the road, and really, not tons of customers are going to have been using these...

We got *lots* of these total.  Order of dozens.

File: pkg/cmd/exec/exec.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	"github.com/spf13/cobra"
+)
+
+// NewExecShim returns a cobra command that shells out the specified target with exactly the same args
+func NewExecShim(name, target string, knownArgs ...string) *cobra.Command {
+	return &cobra.Command{
+		Use:                name,
+		DisableFlagParsing: true,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			execCmd := exec.Command(target, append(knownArgs, args...)...)
```
Comment (smarterclayton) [inline pkg/cmd/exec/exec.go]: You need to execve, not exec directly.

Comment (smarterclayton) [reply pkg/cmd/exec/exec.go]: https://github.com/openshift/origin/blob/master/pkg/cmd/server/start/start_node.go#L394

**Pull request closed without merging**